packages/yew/src/html/component/children.rs

```diff
@@ -186,6 +186,28 @@ where
         // This way `self.iter().next()` only has to clone a single node.
         self.children.iter().cloned()
     }
+
+    /// Convert the children elements to another object (if there are any).
+    ///
+    /// ```
+    /// # let children = Children::new(Vec::new());
+    /// # use yew::{classes, html, Children};
+    /// children.map(|children| {
+    ///     html! {
+    ///         <div class={classes!("container")}>
+    ///             {children}
+    ///         </div>
+    ///     }
+    /// })
+    /// # ;
+    /// ```
+    pub fn map<OUT: Default>(&self, closure: impl FnOnce(&Self) -> OUT) -> OUT {
```

Comment:
Ideally, we should not release features in a patch release.

This could cause the map method to conflict with the same implemented on a potential trait and render a downstream crate not to be able to compile.

See: https://rust-lang.github.io/rfcs/1105-api-evolution.html#minor-change-adding-any-inherent-items

However, since we are pre-1.0 and the risk is relatively low, I am fine with including it.
We can yank the crate if this happens.